src/cython/gudhi/simplex_tree.pyx

```diff
@@ -79,18 +44,20 @@ cdef class SimplexTree:
 
     # The real cython constructor
     def __cinit__(self):
-        self.thisptr = new Simplex_tree_interface_full_featured()
+        cdef Simplex_tree_interface_full_featured* ptr = new Simplex_tree_interface_full_featured()
+        self.thisptr = <intptr_t>ptr
```

Comment:
Again, does cython let you write that as a single line, without naming the variable `ptr`?

Reply:
I agree. Done on 4fd93dbc06b3020ddd856aed2d8ff37e1d27e517
I do not see any other occurrence.